src/Cardano/CoinSelection/LargestFirst.hs

```diff
@@ -34,15 +34,15 @@ import qualified Data.List as L
 import qualified Data.List.NonEmpty as NE
 import qualified Data.Map.Strict as Map
 
--- | Largest-first input selection policy
+-- | Implements the __Largest-First__ coin selection algorithm.
 largestFirst
     :: forall m e. Monad m
     => CoinSelectionOptions e
     -> NonEmpty TxOut
     -> UTxO
     -> ExceptT (ErrCoinSelection e) m (CoinSelection, UTxO)
 largestFirst opt outs utxo = do
-    let descending = NE.toList . NE.sortBy (flip $ comparing coin)
+    let outsDescending = NE.toList $ NE.sortBy (flip $ comparing coin) outs
```

Comment:
This function was applied multiple times to `outs`, but not to any other values.

Therefore, we can apply it just once and then give the result a meaningful name.